### ISSUE TITLE
Use {}/format instead of % for ngettext strings

### ISSUE THREAD

**Issue by sjohannes** [exaile/exaile#599]
In some languages it can make sense to translate some numbered strings without including the number itself.

For example, currently our Arabic localization does this for ["%d second" / "%d seconds"](https://hosted.weblate.org/translate/exaile/master/ar/?checksum=5f827310961c6764) when `%d` is 0, 1, or 2. Luckily this particular string is unused, because otherwise it would cause an error:

```pycon
>>> 'abc' % 1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
```

These strings should be changed from using percent-based formatting to {}-based formatting (`str.format`) because the latter ignores extraneous args:

```pycon
>>> 'abc'.format(1)
'abc'
```

As part of this, I'd want the existing translations to be updated as well, so that we don't invalidate them all.

**Comment by dkstm95** [comment]
Hi :)
Could you tell me one example file that include this issue ?

**Comment by sjohannes** [comment]
The `%d seconds` example comes from [`xl/formatter.py`](https://github.com/exaile/exaile/blob/50f9337ff3749929d0d69a81ce82a85a6507f42c/xl/formatter.py#L601) and the code looks like this:

    text += ngettext('%d second', '%d seconds', span.seconds) % span.seconds

You can find all affected strings by searching for `ngettext` in the source code. There are only a few of them (4 lines in `xl/formatter.py` and 1 line in `xlgui/preferences/plugin.py`), but the bigger task is updating our existing translations (`po/*.po`). For example, the entry in `ar.po` should change from

```
#: ../xl/formatter.py:601
#, python-format
msgid "%d second"
msgid_plural "%d seconds"
msgstr[0] "%d seconde"
msgstr[1] "%d secondes"
```

to something like

```
#: ../xl/formatter.py:601
msgid "{} second"
msgid_plural "{} seconds"
msgstr[0] "{} seconde"
msgstr[1] "{} secondes"
```

**Comment by dkstm95** [comment]
I am now changing all %d to {}, is this right?
(except for %dd or % dk etc)

**Comment by sjohannes** [comment]
No, this task is just for numbers that are translated with `ngettext`. The ones translated with `gettext` are fine and don't need to be changed.

**Comment by dkstm95** [comment]
Oh, what I mean is %d in .po files.

**Comment by sjohannes** [comment]
Only for the strings that are also changed in the `.py` files.
